### DISCUSSION TITLE
Panda dome

### DISCUSSION THREAD

**Discussion by ObiusMarkus** [X2CommunityCore/xcom2-launcher#318] [Help]
Panda dome antivirus critically alerts me that launcher exe is infected

**Comment by Dragon-32** [comment]
At a guess your AV has seen some behaviour (maybe launching another process) as indicative of malware and so classified AML as malware.

All the AV companies I've dealt with have provided the ability to upload samples as false positives.

You can check AML against other AV engines at VirusTotal too.
